dbschema/Banco_de_dados_do_Alerta_Dengue_create.sql

```diff
@@ -206,7 +205,7 @@ CREATE TABLE "Municipio"."Ovitrampa" (
 
 -- Table: "Municipio"."Tweet"
 CREATE TABLE "Municipio"."Tweet" (
-    "id" BIGSERIAL PRIMARY KEY NOT NULL,
+    "id" BIGSERIAL NOT NULL,
```

Comment:
The primary key is defined [here](https://github.com/fccoelho/AlertaDengueCaptura/pull/8/files#diff-c2e28fbd9d078b9162bab8824798792eR213). It was defined twice before 1138463.